src/main/kotlin/org/jitsi/videobridge/octo/OctoRelayService.kt

```diff
@@ -120,6 +120,11 @@ class OctoRelayService : BundleActivator {
          * The send buffer size for the Octo socket
          */
         private const val OCTO_SO_SNDBUF = 10 * 1024 * 1024
+
+        /**
+         * The version of the octo protocol.
+         */
+        const val OCTO_VERSION = 1;
```

Comment:
Unnecessary semicolon - causes ktlint failure. (I'm surprised you were able to commit this?)

Reply:
I don't get an error at commit time. Do you have a hook set up?

Reply:
Yeah, I do - I guess I forgot it was locally configured.